maquetado/scss/layout.scss

```diff
@@ -0,0 +1,9 @@
+.card-container {
+  margin: 30px 50px;
+}
+
+.techs-container {
+  display: flex;
+  flex-flow: row wrap;
```

Comment:
It's not necessary assign the `flex-direction` propery to `row` because the default value is `row`, only put `flex-wrap: wrap`

Reply:
right!